signing_clients/apps.py

```diff
@@ -139,28 +155,28 @@ def __str__(self):
             if name:
                 entry += b'\n '
         entry += b'\n'
-        entry += b'Digest-Algorithms:%s\n' % force_bytes(algos)
+        entry += b'Digest-Algorithms: %s\n' % force_bytes(algos)
         for algo in order:
             entry += b'%s-Digest: %s\n' % (force_bytes(algo.upper()),
                                            b64encode(self.digests[algo]))
         return entry.decode('utf-8')
 
 
 @python_2_unicode_compatible
-class Manifest(list):
+class Manifest(object):
```

Comment:
I like that! `Manifest` never really behaved like a proper list so this is great.